src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -133,38 +146,53 @@ def _get_scripted_model(self, pt_module, example_inputs=None):
         import torch
         import inspect
 
-        def prepare_example_inputs(inputs, input_signature):
-            is_torch_2 = parse(torch.__version__) >= parse("2.0.0")
+        def process_dict_inputs(inputs, input_signature, model):
```

Comment:
I think it makes sense to make a comment in what cases it makes sense to create wrapper for the model

Reply:
added